seed/ts-sdk/accept-header/README.md

```diff
@@ -70,6 +70,15 @@ try {
 If you would like to send additional headers as part of the request, use the `headers` request option.
```

Comment:
`use the `headers` request option.`=> `use the `headers` client or request option.`